sdk/apimanagement/Azure.ResourceManager.ApiManagement/tests/Scenario/ApiManagementServiceResourceTests.cs

```diff
@@ -0,0 +1,318 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.ObjectModel;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.ApiManagement.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.ApiManagement.Tests
+{
+    public class ApiManagementServiceResourceTests : ApiManagementManagementTestBase
+    {
+        public ApiManagementServiceResourceTests(bool isAsync)
+                    : base(isAsync)//, RecordedTestMode.Record)
+        {
+        }
+
+        // For playback
+        private async Task<ApiManagementServiceResource> GetApiManagementServiceAsync()
+        {
+            var resourceGroup = await DefaultSubscription.GetResourceGroups().GetAsync("sdktestrg");
+            var collection = resourceGroup.Value.GetApiManagementServices();
+            return (await collection.GetAsync("sdktestapi")).Value;
+        }
+
+        [Test]
+        public async Task ApplyNetworkConfigurationUpdates()
+        {
+            // Please create the resource first.
+            var apiManagementService = await GetApiManagementServiceAsync();
+            var networkConfigurationContent = new ApiManagementServiceApplyNetworkConfigurationContent();
+            // Test API is in Updating State
+            Assert.ThrowsAsync<Azure.RequestFailedException>(async () => await apiManagementService.ApplyNetworkConfigurationUpdatesAsync(WaitUntil.Completed, networkConfigurationContent));
```

Comment:
Did we create the resource here? Or this comment means that to run this test case, you must create this using some other approcahes (like on portal)?